getting-started/kubearmor_vm.md

```diff
@@ -4,8 +4,10 @@ This recipe explains how to use KubeArmor directly on VM/Bare-Metal host and was
 
 ## Download and Install KubeArmor
 
-1. Download the [latest release of KubeArmor](https://github.com/kubearmor/KubeArmor/releases)
-2. Install KubeArmor `sudo apt install ./kubearmor_${VER}_linux-amd64.deb` ... where VER is the kubearmor release version. This will automatically install the required dependencies.
+1. Install latest bcc from [source](https://github.com/iovisor/bcc/blob/master/INSTALL.md#source), based on the linux distribution you are using.`version 0.21.0 or greater`
```

Comment:
We are yet to make a new release, but after our next release bpfcc-tools are automatically installed in our deb package! We would still need folks on ubuntu 18.04 to install BCC from source but I wouldn't suggest adding it as a step. Rather it would be better add a note/blockquoute after the second point stating 

> We automatically install `bpfcc-tools` with our package, your distribution might have an older version of BCC so consider installing BCC from source in case of errors...

Can improve the wording. But something in line with this is perhaps better imo.